app/models/decidim/census_connector/person.rb

```diff
@@ -12,6 +12,30 @@ class Person
       delegate :id, to: :address_scope, prefix: true
       delegate :id, to: :document_scope, prefix: true
 
+      DOCUMENT_TYPES = %w(dni nie passport).freeze
```

Comment:
Just a thought: all this enums should match the census enums, so it's kind of an API thing. :thinking:
Anyway, if this way is simpler let's leave it as is here

Reply:
Yeah, I had that same thought too, but the code still looks clearer this way to me.

We're not asking those enums to the API but relying on API documentation for these definitions. I guess it could happen that census adds internal states that aren't necessary by this plugin, or things like that. That's why I think it feels nicer if the `API::Person` class is restricted to actual interaction with the API.